### PR TITLE
Automated Test | LPS-190771, LPS-190807

### PR DESCRIPTION
**Pull request by evanilson-santana**
https://liferay.atlassian.net/browse/LPS-190771
https://liferay.atlassian.net/browse/LPS-190807

### CODE REVIEW

**Comment by evanilson-santana** [comment]
ci:test:sf

File: modules/apps/object/object-test/src/testFunctional/tests/ObjectClientExtensions.testcase [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+			ObjectPortlet.addSingleFieldEntryViaUI(entry = "objectEntryManager Entry");
+
+			ObjectAdmin.goToCustomObject(objectName = "ProxyCustomObject");
+
+			AssertElementPresent(
+				key_entry = "objectEntryManager Entry",
+				locator1 = "ObjectPortlet#ENTRY_VALUE");
+		}
+
+		task ("And then the user is able to update entries") {
```
Comment (timpak) [inline modules/apps/object/object-test/src/testFunctional/tests/ObjectClientExtensions.testcase:87]: and Then

File: modules/apps/object/object-test/src/testFunctional/tests/ObjectClientExtensions.testcase [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+				value1 = "objectEntryManager Entry Updated");
+
+			PortletEntry.save();
+
+			AssertElementPresent(
+				key_entry = "objectEntryManager Entry Updated",
+				locator1 = "ObjectPortlet#ENTRY_VALUE");
+		}
+
+		task ("And then the user is able to delete entries") {
```
Comment (timpak) [inline modules/apps/object/object-test/src/testFunctional/tests/ObjectClientExtensions.testcase:101]: and Then

**Comment by evanilson-santana** [comment]
ci:forward

**Pull request closed without merging**